projects/vdk-plugins/vdk-ingest-http/tests/test_ingest_over_http.py

```diff
@@ -226,3 +229,31 @@ def arg(a):
         cert="cert.pem",
         verify=True,
     )
+
+
+@mock.patch("json.dumps")
+def test_ingest_over_http_json_dumps_parameters_propagation(mock_jsondumps):
+    allow_nan = True
+
+    def configuration_side_effect(*args, **kwargs):
+        def arg(a):
+            return args[0] == a
+
+        if arg("INGEST_OVER_HTTP_ALLOW_NAN"):
+            return allow_nan
+        return None
+
+    job_context = MagicMock()
+    job_context.core_context.configuration.get_value.side_effect = (
+        configuration_side_effect
+    )
+    try:
```

Comment:
Is this try-except actually needed? If you expect an exception to be raised, wouldn't it be better to use `with pytest.raises()`?

Reply:
well, with using pytest.raises() - I would've wanted to ensure the exception is raised (requirement);
since exception-or-not does not matter for the particular test, I decided to wrap it with try - so either way the test proceeds to assert_called_with, that is the actual test

Reply:
Ok, I'll give it a pass, though I am a bit concerned with the use of try-except in tests, when we don't know if a method call would throw an exception, or not.